androidApp/build.gradle.kts

```diff
@@ -25,11 +27,47 @@ android {
             excludes += "/META-INF/{AL2.0,LGPL2.1}"
         }
     }
+
+    signingConfigs {
+        create(BuildTypes.RELEASE) {
+            storeFile = file("../config/release.keystore")
+            storePassword = keystoreProperties.getProperty("KEYSTORE_PASSWORD") as String
+            keyPassword = keystoreProperties.getProperty("KEY_PASSWORD") as String
+            keyAlias = keystoreProperties.getProperty("KEY_ALIAS") as String
+        }
+
+        getByName(BuildTypes.DEBUG) {
+            storeFile = file("../config/debug.keystore")
+            storePassword = "F8@k7uen"
```

Comment:
These secrets should be kept in `buildKonfig.properties` or somewhere else that no one can read freely.

Reply:
For `debug.keystore`, it doesn't actually matter. So let's keep it here for simplicity.